symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/pagination/model/RangePaginationAttribute.java

```diff
@@ -0,0 +1,37 @@
+package com.symphony.bdk.core.service.pagination.model;
+
+import lombok.Getter;
+import org.apiguardian.api.API;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Pagination Attribute model to be used in default pagination methods provided by bdk services class.
+ */
+@Getter
+@API(status = API.Status.STABLE)
+public class RangePaginationAttribute {
```

Comment:
`CursorPaginationAttribute`

Reply:
So we have 3 different pagination styles? arghhhhh

Reply:
I also don't like the fact we have 2 different types of pagination in agent endpoints